blueprints/20-tasks/blueprint.py

```diff
@@ -10,17 +10,17 @@
 __email__ = "rocky.craig@hpe.com, zakhar.volchak@hpe.com"
 
 
+import debian
```

Comment:
I actually would like that instead of "import debian" and then use the whole pass, but.. then to keep it relatively consistent, should we "import as" for tmms.utils as well?